lib/instrumentation.js

```diff
@@ -27,16 +27,18 @@ module.exports = function initialize(shim, Koa) {
 }
 
 function wrapMiddleware(shim, middleware) {
-  var router = middleware.router
+  const router = middleware.router
+
   if (router && router.stack && router.stack.length) {
-    var stack = router.stack
-    for (var i = 0; i < stack.length; ++i) {
-      var layer = stack[i]
-      var spec = {
+    const stack = router.stack
+    for (let i = 0; i < stack.length; ++i) {
+      const layer = stack[i]
+      const spec = {
         route: layer.path,
         type: shim.MIDDLEWARE,
         next: shim.LAST,
         promise: true,
+        appendPath: false,
```

Comment:
I'm surprised doing this across the board for all routers isn't causing issues, such as with `koa-route` tests. Our coverage for those tests seems reasonable though. 🤷‍♀️

Reply:
I don't think `koa-route` actually uses a `router` prop, so it probably wouldn't touch this. That instrumentation is pretty simple by comparison.